pkg/reconciler/openshift/tektonconfig/extension.go

```diff
@@ -65,10 +65,11 @@ func (oe openshiftExtension) PreReconcile(ctx context.Context, tc v1alpha1.Tekto
 	}
 
 	r := rbac{
-		kubeClientSet:     oe.kubeClientSet,
-		operatorClientSet: oe.operatorClientSet,
-		ownerRef:          configOwnerRef(tc),
-		version:           os.Getenv(versionKey),
+		kubeClientSet:                    oe.kubeClientSet,
+		operatorClientSet:                oe.operatorClientSet,
+		version:                          os.Getenv(versionKey),
+		disableRbacResourcesForOpenshift: config.Spec.Params,
+		tektonConfig:                     config,
 	}
 	return r.createResources(ctx)
```

Comment:
what if here we check if the `createRbacResources` param is true or not?
if false 
  we get the name of installer set and delete it
if true
  then create an installer set and pass the owner ref of installer set to the create func
  
 ---- 
 inside it as we just need to update all the func 
 if resource exist then update the resource the owner ref  
 
 as it is an upgrade we will get all namespace in the existing `version != 1.6` call